wildfly/configuration/xml/mysql-module.xml

```diff
@@ -14,7 +14,7 @@
 -->
 <module xmlns="urn:jboss:module:1.0" name="com.mysql.jdbc">
   <resources>
-    <resource-root path="mysql-connector-java-5.1.18.jar"/>
+    <resource-root path="mysql-connector-java-5.1.44.jar"/>
```

Comment:
I've not actually run this, but I think you are not downloading it as `mysql-connector-java-5.1.44.jar`, but as `mysql-connector-java.jar` instead, hence this reference would be invalid - no?

Reply:
I see, that's true. This is related with your other comment above, for more explicitness it's good to have the version hardcoded, however we would have the version in 3 different places, making updates a bit "harder". This very PR, with 3 places to change, I made already a couple of mistakes. Do you then still agree with having the version hardcoded?

Reply:
yes, version numbers are extremely important metadata - at least to me

Reply:
done